### ISSUE TITLE
Add readonly=True in related fields

### ISSUE THREAD

**Issue by oscars8a** [aitorbouzas/helpdesk#59]
Hi there,
I think It is necessary to add readonly attribute in related fields because It could generate errors.

https://github.com/aitorbouzas/helpdesk/blob/2a47dceb57b6480e68c8a0d1cec53d34aeea6679/helpdesk/models/helpdesk_ticket.py#L37

**Comment by aitorbouzas** [comment]
You are right, make a PR but remember this is not the correct repository now. Go to https://github.com/aitorbouzas/OCA-helpdesk
Also remember that in v12 `readonly=True` is by default in related fields.

**Comment by oscars8a** [comment]
Thanks for the advice 👌
